src/test/FuzzerImpl.cpp

```diff
@@ -84,6 +86,97 @@ getAndIncrementSequenceNumber()
     return nextSeqNum++;
 }
 
+size_t const MAX_STORED_LEDGER_KEYS = 64U;
+
+size_t numStoredLedgerKeys = 0;
```

Comment:
I temporarily went to `std::vector`, but now it's back to `std::array` because I took your suggestion below for the further simplification of using a single byte to index into a pre-generated array.  Now that the set of stored `LedgerKey`s is pre-generated and fixed-size (always full), I think it's good as a `std::array`.